pennylane/templates/layers/strongly_entangling.py

```diff
@@ -235,3 +235,41 @@ def shape(n_layers, n_wires):
         """
 
         return n_layers, n_wires, 3
+
+    # pylint:disable = no-value-for-parameter
+    @staticmethod
+    def compute_plxpr_decomposition(
+        weights, *wires, ranges=None, imprimitive=None
+    ):  # pylint: disable=arguments-differ
+        wires = qml.math.array(wires, like="jax")
+        ranges = qml.math.array(ranges, like="jax")
```

Comment:
```suggestion
        ranges = qml.math.array(ranges, like="jax") if ranges is not None else None
```

Does it make sense to check for `None`, since `None` is the default value?

Reply:
Actually `ranges` and `imprimitive` should never be `None` here - they are set during `init` if they are not provided. Here I've removed the default value in the arguments for `compute_plxpr_decomposition`, but I'll test with/without these arguments in the tests.